ArmPkg/Library/ArmMmuLib/AArch64/ArmMmuLibPageTableAlloc.c

```diff
@@ -12,7 +12,7 @@
 #include <Library/MemoryAllocationLib.h>
 #include <Protocol/ArmPageTableMemoryAllocation.h>
 
-PAGE_TABLE_MEM_ALLOC_PROTOCOL  *mPageTableMemAllocProtocol = NULL;
+PAGE_TABLE_MEM_ALLOC_PROTOCOL  *PageTableMemAllocProtocol = NULL;

```

Comment:
I think we still want to stick with the m* prefix. But maybe we change it to a different name?